libs/service/constants/src/index.ts

```diff
@@ -3,34 +3,29 @@ import { CENNZNetNetwork } from "@cennznet/api/types";
 
 export const CENNZ_NETWORK = {
 	local: {
-		ChainSlug: "local",
+		ChainSlug: "local" as CENNZNetNetwork,
 		ChainName: "Local Testnet",
 		ExplorerUrl: "https://localhost",
 	},
 	rata: {
-		ChainSlug: "rata",
+		ChainSlug: "rata" as CENNZNetNetwork,
 		ChainName: "Rata Testnet",
 		ExplorerUrl: "https://rata.uncoverexplorer.com",
 	},
 
 	nikau: {
-		ChainSlug: "nikau",
+		ChainSlug: "nikau" as CENNZNetNetwork,
 		ChainName: "Nikau Testnet",
 		ExplorerUrl: "https://nikau.uncoverexplorer.com",
 	},
 
 	azalea: {
-		ChainSlug: "rata",
+		ChainSlug: "rata" as CENNZNetNetwork,
 		ChainName: "CENNZnet Mainnet",
```

Comment:
is this temporary for testing or should it be `azalea`?

Reply:
good spotting, fixed